src/App.tsx

```diff
@@ -1,156 +1,371 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+/* eslint-disable no-console */
+import React, {
+  useEffect, useState, FormEvent, useMemo,
+} from 'react';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { client } from './utils/client';
 
-const USER_ID = 0;
+const USER_ID = 10377;
+
+function getRandomNumber(): number {
+  return Math.floor(Math.random() * 1001);
+}
+
+enum SortType {
+  All = 'All',
+  Active = 'Active',
+  Completed = 'Completed',
+}
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todo, setTodo] = useState<Todo[]>([]);
+  const [selectedTab, setSelectedTab] = useState('All');
```

Comment:
```suggestion
  const [selectedTab, setSelectedTab] = useState(SortType.All);
```
Use Enum instead of hardcoded values